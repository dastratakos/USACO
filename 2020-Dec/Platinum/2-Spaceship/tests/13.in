20 16 20
01011011111111101011
01111110111101101111
11101111101011110110
11111101011110001110
11011000111010111001
00011111110011101111
11111010010001111100
11001110110101101110
10100010111000110010
10100010010001001111
00011010011100100100
01110110011011011111
00000101011010100101
01000011110110001010
10101101110101100110
11100101111111111101
11111000111110101111
01100110010111001110
10011111110110110100
01111101111011111011
3 6 3 18
5 11 11 2
1 3 1 18
5 1 4 1
1 9 4 17
1 11 1 19
5 16 5 13
2 14 4 19
3 19 1 12
3 10 4 4
2 10 5 20
1 8 4 16
5 19 2 17
2 15 3 9
4 14 3 13
2 15 1 20
1 2 5 9
4 8 1 18
8 18 1 2
4 3 4 3
