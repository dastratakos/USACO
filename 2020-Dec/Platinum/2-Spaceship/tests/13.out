640450293
886313896
689831991
494788119
570429944
222941461
146352620
362404624
1096616
121348005
432018387
454815245
666458903
61987807
396357234
105928332
515814917
467072163
930259540
715530412
